src/output/index.test.js

```diff
@@ -28,6 +29,32 @@ test('outputs to stderr', () => {
   expect(stdmock.flush().stderr).toEqual(['it works'])
 })
 
+describe('timestamps', () => {
+  let origFormat = moment.prototype.format
+  beforeAll(() => {
+    process.env.HEROKU_TIMESTAMPS = '1'
+    const timestamps = jest.fn()
+    timestamps.mockReturnValue('2017-06-19T07:42:57-07:00')
+    moment.prototype.format = timestamps
+  })
+
+  afterAll(() => {
+    delete process.env['HEROKU_TIMESTAMPS']
```

Comment:
So one thing, we should use "bin name" logic like we used for `HEROKU_DEBUG`/`CLI_ENGINE_DEBUG`. It's in cli-engine-config (can dig it up if you can't find it)

we've done this a few times now, but I think a better way to do this is like this:

```js
// at the top of the file for all test (so long as it doesn't break anything)

let env = process.env
beforeEach(() => {
  process.env = {}
})

afterEach(() => {
  process.env = env
})

// then in the test
beforeEach(() => {
  process.env.HEROKU_TIMESTAMPS = '1'
})
```

this way it 'resets' the env vars every time the tests are run so the local env vars can't screw anything up and everything always gets reset back after they're done